apps/emqx/src/emqx_schema.erl

```diff
@@ -1294,6 +1298,49 @@ fields("listener_quic_ssl_opts") ->
     );
 fields("ssl_client_opts") ->
     client_ssl_opts_schema(#{});
+fields("ocsp") ->
+    [
+        {"enable_ocsp_stapling",
+            sc(
+                boolean(),
```

Comment:
there had been some discussion about abusing `enable` flag in many places.
is it really necessary for ocsp ?
e.g. if the parent field is set to `null` (`undefined` from HOCON map), doesn't that serve as disable ?

Reply:
This struct is always present because of the fields with default values, right?

We _could_ interpret the absence of `responder_url` as a sign that it's disabled, but in my opinion being explicit with an `enable` flag is more clear than it being implicitly disabled by the absence of a certain field.  :thinking: 

If we want to avoid using explicit configs, then we can use the absence of that field to disable it, yes.